geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -1,17 +1,27 @@
 {% load i18n static corpus_extras %}
-<section id="itt-panel">
+{% if perms.corpus.change_document and document.iiif_urls and not edit_mode %}
```

Comment:
I was wondering about that—it at least needs to have images, right? Since our SAS and Tahqiq plugins require Annotorious, which in turn requires OSD/image.

Reply:
Technically annotation only requires a canvas. I can't remember what I've tested with annotorious but I know Mirador works fine with canvases that don't have images. We may need to provide a placeholder image to make annotorious happy, but we def. need to support transcription for documents without images.

Reply:
Ah, I see! So in the case of fragments without images, do we need to check for the presence of a canvas/manifest in order to launch Tahqiq? And would OSD receive some other tile source, something like `Fragment.manifest.canvases`?

Reply:
I was thinking we would use/generate canvas ids from our local geniza manifests. But I can't remember if I've documented that anywhere! I think that it isn't an mvp editor feature, since we want to handle the easy cases first — but we will def. have existing transcriptions without images that we'll need to migrate and display, so we'll have to have a convention for local canvas identifiers for those.

If it seems simpler to limit the editor to documents with iiif images for now, we can do that and add a comment noting that it is temporary.